<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullSrceenWatch</class>
 <widget class="QMainWindow" name="FullSrceenWatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2,1,2,1">
    <item>
     <widget class="QWidget" name="dayYearWidget" native="true">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="yearLcd">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255)</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="dayLcd">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 255);</string>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="hourLcd">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 36pt &quot;04b_21&quot;;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="colonLcd1">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="minLcd">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 36pt &quot;04b_21&quot;;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="colonLcd2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="secLcd">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="msecLcd">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>161</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>6</number>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
